DataFormats/TrackReco/interface/TrackExtraBase.h

```diff
@@ -73,12 +78,15 @@ class TrackExtraBase
 
     }
 
+    TrajParams const & trajParams() const  {return m_trajParams;}

```

Comment:
But at the moment you have the member data marked as `transient` in the classes_def.xml file. If it stays that way I think we need to document it in the header.